src/components/date/__internal__/navbar/navbar.component.tsx

```diff
@@ -1,23 +1,27 @@
 import React from "react";
-import PropTypes from "prop-types";
 import StyledButton from "./button.style";
 import StyledNavbar from "./navbar.style";
 import Icon from "../../../icon";
 
-const Navbar = ({ onPreviousClick, onNextClick, ...props }) => (
+export interface NavbarProps {
+  onPreviousClick?: () => void;
+  onNextClick?: () => void;
+  className?: string;
+}
+
+export const Navbar = ({
+  onPreviousClick,
+  onNextClick,
+  ...props
+}: NavbarProps) => (
   <StyledNavbar {...props}>
-    <StyledButton onClick={() => onPreviousClick()}>
+    <StyledButton onClick={() => onPreviousClick?.()}>
```

Comment:
```suggestion
    <StyledButton onClick={onPreviousClick}>
```

**suggestion** (non-blocking): any good reason why we can't simply do this?

Same for the "next" button of course

Reply:
hmm, I've looked in the Cypress tests for Date and can't find any that look like they do this - but I'll take your word for it. Happy to accept the change for now but really this is saying to me that we should change that Cypress test. Presumably when the Cypress test is updated to TS it will have to update anyway, as you can't call an optional function prop!